examples/sw/benchmarks/README.md

```diff
@@ -62,6 +62,32 @@ seedcrc          : 0xe9f5
 Correct operation validated. See README.md for run and reporting rules.
 ```
 
+### Choice of ISA string
+
+Different ISAs (to choose different RISC-V ISA extensions) can be selected by
+passing the desired ISA string into `RV_ISA` when invoking make.
+
+```
+make -C ./examples/sw/bencharmsk/coremark clean
+make -C ./examples/sw/benchmarks/coremark RV_ISA=rv32imc
+```
+
+This will build CoreMark using the 'C' extension (compressed instructions).
+
+If CoreMark is already built a `clean` must be done before building
+for a new ISA string to ensure a full rebuild.
```

Comment:
Since I've started with the nitpicks... This sentence is quite hard to read. Maybe simpler: "When changing `RV_ISA`, you must clean out any old build with `make clean` and rebuild."

Also, you changed "CoreMark" to "Coremark" at the top of the file. Which capitalisation is correct?